Test failure with gnome-keyring on Ubuntu Xenial

From the current master:
```
$ go test github.com/tmc/keyring
Get() error: No such interface 'org.freedesktop.Secret.Collection' on object at path /org/freedesktop/secrets/collection/Default
--- FAIL: TestBasicSetGet (0.00s)
	keyring_test.go:20: Set() error: keyring/dbus: CreateItem error: No such interface 'org.freedesktop.Secret.Collection' on object at path /org/freedesktop/secrets/collection/Default
	keyring_test.go:24: Get() error: No such interface 'org.freedesktop.Secret.Collection' on object at path /org/freedesktop/secrets/collection/Default
FAIL
FAIL	github.com/tmc/keyring	0.009s
```

The problem seems to be that by default, the plugin is looking for a password collection called "Default" rather then looking at which password chain _is_ the default (on my system it defaults to being named "Login").

--- a/keyring_linux.go
+++ b/keyring_linux.go
@@ -10,7 +10,7 @@ import (
 const (
 	ssServiceName     = "org.freedesktop.secrets"
 	ssServicePath     = "/org/freedesktop/secrets"
-	ssCollectionPath  = "/org/freedesktop/secrets/collection/Default"
+	ssCollectionPath  = "/org/freedesktop/secrets/aliases/default"
 	ssServiceIface    = "org.freedesktop.Secret.Service."
 	ssSessionIface    = "org.freedesktop.Secret.Session."
 	ssCollectionIface = "org.freedesktop.Secret.Collection."

